packages/wallet-sdk/src/provider/CoinbaseWalletProvider.test.ts

```diff
@@ -702,8 +723,9 @@ describe("CoinbaseWalletProvider", () => {
           method: "wallet_addEthereumChain",
           params: [{}],
         });
-      }).rejects.toThrowError(
-        '"code" must be an integer such that: 1000 <= code <= 4999',
```

Comment:
this was wrong:
this test was to verify whether the "chainName is a required field" error was returned. 
however, our provider was using an invalid error code, `0`, which caused the error message to be an internal error without providing any useful information to users.